src/components/scenes/PasswordLoginScene.tsx

```diff
@@ -91,7 +91,6 @@ export const PasswordLoginScene = (props: Props) => {
     string | undefined
   >(undefined)
   const [password, setPassword] = React.useState('')
-  const [spinner, setSpinner] = React.useState<boolean>(false)
```

Comment:
Why are we dropping the spinner? If you hit "enter" on the keyboard, is there any feedback while the login takes place, which can be many seconds?

Reply:
Not sure how this got removed, will re-add

Reply:
Ah yeah I thought all the spinner handling was moved to the button but I forgot about the text inputs.

I made a change also to disable the other buttons (create acc, forgot pw) while spinning either from "enter" on keyboard or if submit was tapped.